src/VisualStudio/VisualBasic/Impl/LanguageService/VisualBasicPackage.vb

```diff
@@ -69,7 +69,7 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic
 
             packageInitializationTasks.AddTask(
                 isMainThreadTask:=False,
-                task:=Function(packageInitializationTasks2, cancellationToken) As Task
+                task:=Function() As Task
```

Comment:
Oh that's right I forgot VB lets you do this. I figured remove it from the C# side too....